docs/sections/user_guide/yaml/components/make_hgrid.rst

```diff
@@ -0,0 +1,24 @@
+.. _make_hgrid_yaml:
+
+make_hgrid
+==========
+
+Structured YAML to run :ufs-utils:`make_hgrid<make-hgrid>` is validated by JSON Schema and requires the ``make_hgrid:`` block, described below. If ``make_hgrid`` is to be run via a batch system, the ``platform:`` block, described :ref:`here <platform_yaml>`, is also required.
+
+Here is a prototype UW YAML ``make_hgrid:`` block, explained in detail below:
+
+.. highlight:: yaml
+.. literalinclude:: ../../../../shared/make_hgrid.yaml
+
+UW YAML for the ``make_hgrid:`` Block
+-------------------------------------
+
+execution:
+^^^^^^^^^^
+
+See :ref:`here <execution_yaml>` for details.
+
+run_dir:
+^^^^^^^^
+
+The path to the directory where ``make_hgrid`` will write its outputs.
```

Comment:
Sorry if I missed it, or if it's still a work in progress, but: The schema seems to permit (and conditionally require) many more keys than just these two. I don't think we should re-document them here if there's documentation we can point to, but maybe we should explicitly say that more is possible than what's documented here, and point to the authoritative docs. If our key names differ from what's in the authoritative docs, we could perhaps give a table here matching each UW YAML key name to the name used in those docs so that the user could understand the meaning of the setting and the kinds of values expected.

Reply:
Yeah, thats what I was thinking in addition to adding comparisons for the existing YAMLs that we're replacing, working on it

Reply:
Added quite a lot more key details here, let me know what you think of the formatting